scripts/install-helpers/baremetal-coco/install.sh

```diff
@@ -8,6 +8,19 @@ KERNEL_CONFIG_MC_FILE="./96-kata-kernel-config-mc.yaml"
 SKIP_NFD="${SKIP_NFD:-false}"
 TRUSTEE_URL="${TRUSTEE_URL:-"http://kbs-service.trustee-operator-system:8080"}"
 CMD_TIMEOUT="${CMD_TIMEOUT:-900}"
+CLUSTER_HTTPS_PROXY="$(oc get proxy/cluster -o json | jq .spec.httpsProxy)"
+CLUSTER_NO_PROXY="$(oc get proxy/cluster -o json | jq .spec.noProxy)"
+
+[ "$CLUSTER_HTTPS_PROXY" = "null" ] && CLUSTER_HTTPS_PROXY=""
+[ "$CLUSTER_NO_PROXY" = "null" ] && CLUSTER_NO_PROXY=""
```

Comment:
This should be set after we have checked for oc and jq.
I think, it's better to make jq also mandatory. So you can move 'check_command jq' after "check_command oc", rename set_aa_kbc_params_for_kata_agent to something like set_kernel_params_for_kata_agent and set the proxy.

Reply:
without dependencies to external tool you can `oc get proxy/cluster -o jsonpath={.spec.httpsProxy}'`

Reply:
I can see now that `jsonpath` is widely used below so perhaps just make consistent?

Reply:
I've reworked this bit, please, see the latest push.